sdk/spring-boot.html

```diff
@@ -0,0 +1,154 @@
+<!doctype html>
+<html lang="en" class="h-100">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="This organization develops open source software for Open Digital Architecture">
+    <meta name="keywords" content="ODA, Open Digital Architecture, ODA Component SDKs, ODA Component RIs, ODA Canvas SDKs, ODA Canvas RIs">
+    <meta name="author" content="ODA">
+    <meta name="generator" content="Hugo 0.88.1">
+    <title>Open Digital Architecture</title>
```

Comment:
The title should be related to SDK

Reply:
Actually you can ignore this comment - it will change in separate PR for all pages. So, up to you - you can add it in the scope of current changes or it will be done separate.